Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/mysql/MySQLModule.java

```diff
@@ -26,12 +27,18 @@
     public MySQLModule(final Node parent) {
         super(MYSQL_DATABASE_MODULE_ID, MODULE_NAME, parent);
         createListener();
+
+        AzureEventBus.after("mysql|server.create", this::onMySqlServerCreatedOrRemoved);
+        AzureEventBus.after("mysql|server.delete", this::onMySqlServerCreatedOrRemoved);
+    }
```

Comment:
how about replace "this::onMySqlServerCreatedOrRemoved" with "e -> this.refreshItems()" ?